cmd/ssh.go

```diff
@@ -20,59 +20,68 @@ import (
 )
 
 const sshDescription = `
-For any nodes created through Rancher using docker-machine, 
+For any nodes created through Rancher using docker-machine,
 you can SSH into the node. This is not supported for any custom nodes.
 
 Examples:
 	# SSH into a node by ID/name
 	$ rancher ssh nodeFoo
 
+	# SSH into a node by ID/name using the external IP address
+	$ rancher ssh -e nodeFoo
+
 	# SSH into a node by name but specify the username to use
-	$ rancher ssh -l user1 nodeFoo
+	$ rancher ssh -u user1 nodeFoo
```

Comment:
This flag should stay as is, it matches ssh usage.

Reply:
Ok, I've reverted that and made a new PR